### PR TITLE
sass-lint

### PR DESCRIPTION
**Pull request by KeitIG**
Fix #394

### CODE REVIEW

File: src/ui/styles/main.scss [hunk 1]
```diff
@@ -1,72 +1,56 @@
...
 
-/*
-|--------------------------------------------------------------------------
-| Views
-|--------------------------------------------------------------------------
-*/
+@import "view-common";
+@import "view-library";
+@import "view-playlists";
+@import "view-settings";
```
Comment (YurySolovyov) [inline src/ui/styles/main.scss]: Don't we use single quotes in JS?

**Comment by YurySolovyov** [comment]
Have you considered using `stylus` or `postcss` ? I hate sass for native deps

File: src/ui/styles/main.scss (hunk 1 above)
Comment (KeitIG) [reply src/ui/styles/main.scss]: it's a scss file here :p

**Comment by KeitIG** [comment]
@YurySolovyov Yeah, you are right, I like stylus a lot, and someone told me postcss was amazing. I should give them a try.

**Pull request merged**